### ISSUE TITLE
🐛 Different custom fingerprint are reported with the same issueId

### ISSUE THREAD

**Issue by yutaro-biz** [DataDog/browser-sdk#2076]
**Describe the bug**
When using `@datadog/browser-rum` version `4.35.0` (latest at the time of writing), errors has different custom fingerprint are reported with the same issueId.

**To Reproduce**
1. we set up rum this way
``` js
  if (e instanceof HttpErrorResponse) {
      const error = new Error(e.message);
      datadogRum.addError(error, {
        'error.fingerprint': `${e.url}:${e.status}`,
      });
  }
```

2. When looking at the datadog rum application, errors with different fingerprint(for example `https://4csogxtspjtgxxx8gkjuhygpvlj2.requestly.me/400:400` and `https://requestly.dev/api/mockv2/o/400?rq_uid=4csogXTSPjTGXXx8gkjuHYgpVlj2:400`) are reported as same issue.

**Expected behavior**
According to following docs,  we can report errors with different issueId each custom fingerprint.
https://docs.datadoghq.com/real_user_monitoring/error_tracking/custom_grouping

**Comment by bcaudan** [comment]
Hi @yutaro-biz,

Thanks for raising this!
There is indeed an issue on our side with this feature, we'll removed this documentation for now (https://github.com/DataDog/documentation/pull/17297) and we will work on a fix.

We will keep you posted here when we will have something available.

**Comment by bcaudan** [comment]
Hi @yutaro-biz,

We have released a fix with the version `4.42.0`, the doc has not yet been updated but you can have a look at the [linked PR ](#2189) to see how to provide the custom fingerprint.
